.flake8

```diff
@@ -8,6 +8,7 @@ per-file-ignores =
     tests/e2e/utils/data_builder/__init__.py:F401
     tests/e2e/utils/data_fetcher/__init__.py:F401
     aws_lambda_powertools/utilities/data_classes/s3_event.py:A003
+    aws_lambda_powertools/event_handler/openapi/compat.py:F401
```

Comment:
It is better to add this ignore in the `ruff.toml` file.

Reply:
Done!